CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ UNRELEASED
 * [ [#1170](https://github.com/digitalfabrik/integreat-cms/issues/1170) ] Add map preview on POI form
 * [ [#1579](https://github.com/digitalfabrik/integreat-cms/issues/1579) ] Fix auto-filling of coordinates for multiple street numbers
 * [ [#1767](https://github.com/digitalfabrik/integreat-cms/issues/1767) ] Revert statistics calculation to original & add online downloads column
+* [ [#1746](https://github.com/digitalfabrik/integreat-cms/issues/1746) ] Hide analytics section (partially) for author, editor and terminplaner
```

Comment:
```suggestion
* [ [#1746](https://github.com/digitalfabrik/integreat-cms/issues/1746) ] Hide analytics section (partially) for author, editor and event manager
```

Reply:
Modified. I moved this entry to UNRELREASED.